python/cuspatial/cuspatial/utils/traj_utils.py

```diff
@@ -1,3 +1,47 @@
+import numpy as np
+
+from cudf.utils.dtypes import is_datetime_dtype
+
+
+def normalize_point_columns(xs, ys):
```

Comment:
Should this be a utility for more than just trajectories? spatial_window could use this too, I think.

Reply:
Yes that seems appropriate.